frontend/packages/ceph-storage-plugin/src/components/create-storage-system/create-storage-system-steps/capacity-and-nodes-step/capacity-and-nodes-step.tsx

```diff
@@ -237,7 +254,7 @@ const SelectedCapacityAndNodes: React.FC<SelectedCapacityAndNodesProps> = ({
 type SelectedCapacityAndNodesProps = {
   capacity: WizardState['capacityAndNodes']['capacity'];
   storageClassName: string;
-  enableArbiter: WizardState['capacityAndNodes']['enableArbiter'];
+  arbiterLocation: WizardState['capacityAndNodes']['arbiterLocation'];
```

Comment:
Storing `arbiterLocation` since this is the only thing need to be passed to request.
We do not need to store checked/unchecked value as previously.